### PR TITLE
[Component] Switch: Add SwiftUI version

### PR DESCRIPTION
**Pull request by robergro**
This pull request has:
- [x] Code documentation on publics
- [x] Accessibility
	- [x] Dynamic sizes
	- [x] Identifiers
- [x] Tests
- [x] Demo update
- [x] Wiki

The snapshot testing review is [here](https://github.com/adevinta/spark-ios-snapshots/pull/61)

### CODE REVIEW

File: core/Sources/Components/Switch/View/SwiftUI/SwitchView.swift [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+        }
+    }
+
+    // MARK: - Subview Maker
+
+    private func subviewsTypes() -> [SwitchSubviewType] {
+        return (self.viewModel.isToggleOnLeft == true) ? SwitchSubviewType.leftAlignmentCases : SwitchSubviewType.rightAlignmentCases
+    }
+
+    private func makeSubview(from type: SwitchSubviewType) -> some View {
```
Comment (michael-zimmermann) [inline core/Sources/Components/Switch/View/SwiftUI/SwitchView.swift:74]: I think if you add `@ViewBuilder` to the function, you don't need to pack the single views into AnyView.

**Review by michael-zimmermann — COMMENTED** [review]
Looks good, just one small comment

File: core/Sources/Components/Switch/View/SwiftUI/SwitchView.swift (hunk 1 above)
Comment (robergro) [reply core/Sources/Components/Switch/View/SwiftUI/SwitchView.swift:75]: Oh yeah ! Thank you

**Review by michael-zimmermann — APPROVED** [review]
👍

**Pull request merged**